website/docs/onboarding/containers/installing-podman.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 10
+title: Installing Podman
+description: Podman Desktop can assist you to install Podman on Windows and macOS.
+tags: [podman-desktop, podman, installing, windows]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos]
+---
+
+# Installing Podman with Podman Desktop
+
+On Windows and macOS, running the Podman container engine requires running a Linux distribution on a virtual machine.
+Podman Desktop can assist you to install the Podman container engine in a Fedora distribution of Linux, on a virtual machine.
+
+Main benefits are:
+
+- Ease of use.
+- On Windows: Windows Subsystem for Linux version 2 (WSL 2) native virtualization performance.
+
+Consider rather [installing Podman with OpenShift Local](/docs/onboarding/containers/installing-podman-with-openshift-local-on-windows) if:
+
+- You prefer working with a RHEL distribution of Linux.
+- Your environment doesn't allow you to meet the prerequisites.
+
+#### Prerequisites
+
+- Podman is not installed.
+- 6 GB RAM.
+- The host runs Windows or macOS.
+- Windows prerequisites:
+  - No VPN.
+  - [WSL prerequisites](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed):
+    - User with administrator privileges.
+    - Windows 64bit.
+    - Windows 10 Build 18362 or greater, or Windows 11.
+    - On a virtual machine: [Nested Virtualization enabled](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization#configure-nested-virtualization).
+  - No WSL 2 Linux virtual machine is running.
+
+#### Procedure
+
+1. (On Windows) Enable the WSL feature without installing the default Ubuntu distribution of Linux.
```

Comment:
Sorry if a dupe, it seems GitHub ate this comment earlier - see https://github.com/containers/podman-desktop/pull/4209 - Luca has it so if the onboarding pre-requisite check doesn't see WSL2 it offers to install. It seems worth mentioning? But I am not sure what is the better experience, for users to set it up themselves and we use the pre-requisite check to catch a few who missed that step, or just let all users rely on it?

Reply:
Yes we discussed that and the decision was:
- Keep now.
- Remove once new onboarding that automates WSL installation is out.